scripts/zones/Bostaunieux_Oubliette/npcs/Novalmauge.lua

```diff
@@ -88,8 +90,9 @@ function onTrigger(player,npc)
         player:startEvent(0x000b);
     elseif (theRumor == QUEST_COMPLETED) then
         player:startEvent(0x000e); -- Standard dialog after "The Rumor"
+		player:setVar("theHolyCrestCheck",0);
```

Comment:
you have 2 tabs here. this project uses indents made of spaces. configure your editor to use spaces in place of tabs 4 at a  time (nobody ever actually sits there tapping spacebar and counting it, we just make our tabkeys actually create the correct number of spaces)

Since this line is 2 indents deep, you need 8 spaces here.

Reply:
actually I'm not sure this line even belongs here. you've zero'd it out in more than one spot.

Reply:
Ok, editor configured... =/ 
and the second zero is there to make sure you are referred back to the reminder to finish the DRG quest if it is flagged.